parquet/src/test/java/org/apache/iceberg/avro/TestParquetReadProjection.java

```diff
@@ -45,4 +45,20 @@ protected GenericData.Record writeAndRead(
 
     return Iterables.getOnlyElement(records);
   }
+
+  // Empty struct read is not supported for Parquet
+  @Override
+  public void testEmptyStructProjection() throws Exception {}
```

Comment:
These tests was added for Avro. When empty struct was supported. But it is not yet supported for parquet. Since both avro and parquet now use the common class. Ignoring the tests for parquet.

Reply:
Also, This class uses (extends) `TestReadProjection` from api module now. Instead of duplicated local copy.